qa/lsan.supp

```diff
@@ -0,0 +1,18 @@
+# leak suppressions needed to run vstart.sh with WITH_ASAN=ON
+# to enable, add this to the environment:
+# LSAN_OPTIONS="suppressions=../qa/lsan.supp"
+
+# from perfglue/heap_profiler.cc
+# gperftools allocates a singleton and never frees it
+leak:^MallocExtension::Register
+
+# from src/ceph.in
+# leaks are specific to older python versions, and go away in 3.12
+leak:^PyObject_Malloc
+leak:^_PyObject_Realloc
+leak:^PyMem_Calloc
+leak:^PyMem_Malloc
+leak:^PyUnicode_New
+leak:^_PyBytes_FromSize
+leak:^_PyObject_GC_Alloc
+leak:^_PyObject_GC_Resize
```

Comment:
this time the correct functions that should be added when running with rhel8 (Python 3.6.8):
```
leak:^_PyObject_Alloc
leak:^PyType_GenericAlloc
leak:^PyBytes_FromStringAndSize
leak:^set_table_resize
```